src/base/orgs/View/Projects.svelte

```diff
@@ -55,7 +56,9 @@
             </span>
           </Widget>
         {:else} <!-- Anchored project -->
-          <Widget {project} org={org.address} {config} />
+          <Widget {project} org={org.address} {config}>
+            <span slot="stateHash">commit {project.anchor.stateHash}</span>
+          </Widget>
```

Comment:
What do these changes do?

Reply:
Ah I see, they allow passing the stateHash from outside.

Reply:
Yes exactly, since the seed node doesn't provide anchors, we should have a way to show Projects without anchors or relation to orgs contracts.
Eventually this could be part of what the seed node provides?